### DISCUSSION TITLE
Got a notification from KeepassDX "Executing the command"

### DISCUSSION THREAD

**Discussion by ghost** [Kunzisoft/KeePassDX#1700] [Q&A] [answered]
Hi, I just noticed an active notification from KeepassDX which said something like "Executing the command". I'm literally scared and uninstalled the app. I will try re-installing. How is KeepassDX even executing an "command" without Internet permission? I downloaded KeepassDX from the Play store I think.

**Comment by ghost** [comment]
An Update: I've reinstalled KeepassDX from the Play store and have not faced this issue again. I don't know how this even happened, since I installed KeepassDX from the Play store I think.

**Reply by J-Jamet** [reply]
It's not an issue, any program executes commands, it's normal and its mainspring.

**Comment by J-Jamet** [comment, marked as answer]
KeePassDX is a program like any other, and a program executes commands you give it all the time. Pressing a button executes a command. Here it's simply a wording to indicate that an action is being performed on the database locally when it's read-only without a save through the URI.
